package.json

```diff
@@ -84,13 +84,14 @@
     "ms": "^2.1.3",
     "parse-function": "^5.6.4",
     "promise-retry": "^1.1.1",
+    "read-package-json": "^5.0.1",
```

Comment:
why do we need this package?

Reply:
well, this one to read the version of any package easier

Reply:
no, I'm against adding extra packages without any real need. 
please consider a workaround